src/test/tests/unit/atts_assign.py

```diff
@@ -0,0 +1,444 @@
+# ----------------------------------------------------------------------------
+#  CLASSES: nightly
+#
+#  Test Case:  atts_assign.py
+#
+#  Tests: Behavior of assignment for attribute objects. Ensures good cases
```

Comment:
Are you going to add tests for Setting colors? too few number, out-of-uchar-range numbers?

Reply:
Yes, I will. That is the one file I have yet to finish here.